pkg/client/generated/management/v3/zz_generated_ecr_credential_plugin.go

```diff
@@ -0,0 +1,14 @@
+package client
```

Comment:
why is this part of the PR?

Reply:
Someone else touched rancher types and forgot to run `go generate` to generate norman types...

Reply:
we should find this someone and educate them.

Reply:
It's a separate commit for that reason :)

Reply:
https://github.com/rancher/rancher/pull/34372/files I did the same on my PR 😆